layouts/_default/baseof.html

```diff
@@ -191,5 +191,26 @@ <h1 data-pagefind-meta="title">About{{ if (isset .Params "subtitle") }} - {{ .Pa
   {{ end }}
 
 </body>
+{{ if .Page.Store.Get "hasGraphviz" -}}
+<script src="{{ relURL "js/viz-global.js" }}"> </script>
+<script type="text/javascript">
+  (() => {
+    let vizInstance
+    [...document.querySelectorAll("pre[class=graphviz]")].forEach(async (x) => {
+      if (!vizInstance) vizInstance = await Viz.instance()
+      const options = {
+        graphAttributes: {
+          bgcolor: "transparent",
+        },
+        engine: x.getAttribute("engine") || "dot",
+      }
+      const svg = vizInstance.renderSVGElement(x.innerText, options)
+      svg.classList.add("graphviz")
+      x.parentNode.insertBefore(svg, x);
+      x.style.display = 'none'
+    });
+  })();
+</script>
```

Comment:
Oh, I guess there's a point to be made about moving this logic to `application.js` and only keep a call to `Graphviz.render()` here, right? I'll do that real quick.